gguf-py/gguf/tensor_mapping.py

```diff
@@ -1059,6 +1059,8 @@ class TensorNameMap:
         MODEL_TENSOR.V_MMPROJ: (
             "multi_modal_projector.linear_{bid}",
             "visual.merger.mlp.{bid}", # qwen2vl
+            "vision_language_adapter.w_in", # pixtral
+            "vision_language_adapter.w_out", # pixtral
```

Comment:
this is potentially be wrong, as we expect a `{bid}` number in the name. Otherwise, you should have a custom renaming from `w_in` to `multi_modal_projector.linear_1` inside the `modify_tensors` function. Same for `w_out`